Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:alpine
+
+# Directory
+ARG APP_DIR=app
+RUN mkdir -p ${APP_DIR}
+WORKDIR ${APP_DIR}
+
+# Install dependencies
+COPY package*.json ./
```

Comment:
Not sure why this is a special step, caching?
Otherwise the install could just go after the COPY and `package.json` would be available.

Reply:
Yeah, caching. Maybe not needed for something like this?

Reply:
Depends on how often you're rebuilding it. The nice thing about this setup is it doesn't need to rebuild dependencies if you modify something in the file below that, but unless you're doing a lot of editing I would say simplicity trumps that gain.